src/components/FormValidator/FormValidator.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export function useFormWithValidation(inputs) {
```

Comment:
that's a nice validation hook


but it's better to put it in folder `hooks` inside file `useFormWithValidation.js`